ofnetAgent.go

```diff
@@ -517,3 +580,498 @@ func (self *OfnetAgent) DummyRpc(arg *string, ret *bool) error {
 	log.Infof("Received dummy route RPC call")
 	return nil
 }
+
```

Comment:
Please see if BPG code can go into its own file and create an independent BGP object